src/components/card/index.tsx

```diff
@@ -1,11 +1,116 @@
-type CardProps = {
+import { useState } from 'react';
+
+import Star from 'assets/icons/star.png';
+import Later from 'assets/icons/later.png';
+import Share from 'assets/icons/share.png';
+import Play from 'assets/icons/bt_play.png';
+import Triangle from 'assets/icons/triangle.png';
+
+export type ItemInterface = {
+  id: number;
   img: string;
+  title: string;
+  description: string;
+};
+
+type CardProps = {
+  index: number;
+  item: ItemInterface;
+  border?: boolean;
+  scale?: boolean;
+  idSelected?: number | undefined;
+  type?: 'wide' | 'large';
+  // eslint-disable-next-line no-unused-vars
+  onSelected: (item: ItemInterface) => void;
 };
 
-const Card = ({ img }: CardProps) => (
-  <div className="card m-0.5 hover:z-10 overflow-hidden transform transition duration-500 hover:scale-110 cursor-pointer">
-    <img src={img} alt="" />
-  </div>
-);
+const Card = (card: CardProps) => {
+  const {
+    onSelected = () => null,
+    scale,
+    type,
+    item,
+    border,
+    idSelected,
+    index,
+  } = card;
+  const [isHover, setHover] = useState(false);
+
+  const printBorder =
+    (border && idSelected === item.id) ||
+    (idSelected === undefined && index === 0);
+  return (
+    <div onClick={() => onSelected(item)}>
+      {type === 'wide' ? (
+        <div
+          className={`flex flex-col items-center card hover:z-10 overflow-hidden  cursor-pointer ${
+            scale && 'transform transition duration-200 hover:scale-110'
+          }`}
+        >
+          <img
+            className={`img-card ${printBorder && 'border-4'}`}
```

Comment:
When printBorder is `false`, it will be visible on the DOM, not always it is an issue but if for some reason you have a class called `false`, it will. There are a lot of ways to fix it but a good alternative is using [classnames](https://www.npmjs.com/package/classnames) package.